### DISCUSSION TITLE
The best distribution I've ever used!

### DISCUSSION THREAD

**Discussion by conree** [SpiralLinux/SpiralLinux-project#325] [General]
I want to thank the developer for creating the best distribution I've ever used. It was trivial to install and has all the features I use on a daily basis. I love that it comes preconfigured to use btrfs and snapper and that it has a reasonable selection of apps for the basic installation, no bloat and lots of flexibility to install the stuff you really need. A job exceptionally well done!

**Comment by geckolinux** [comment]
Really glad to hear that you like it, thanks for saying so!
